<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>results_widget</class>
    <widget class="QWidget" name="results_widget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>608</width>
                <height>191</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(77, 80, 87);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                <widget class="QTableView" name="results_table">
                    <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="sizeAdjustPolicy">
                        <enum>QAbstractScrollArea::AdjustToContents</enum>
                    </property>
                    <attribute name="horizontalHeaderStretchLastSection">
                        <bool>false</bool>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                        <bool>false</bool>
                    </attribute>
                </widget>
            </item>
            <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                <widget class="QPushButton" name="again_button_res">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>10</pointsize>
                        </font>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">QPushButton {
                            color: #f28d01;
                            }

                            :hover{
                            color: #b56900;
                            }
                        </string>
                    </property>
                    <property name="text">
                        <string>Again?</string>
                    </property>
                    <property name="flat">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="again_bottom_spacer_res">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>65</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
